src/main/java/seedu/modtracker/ModTracker.java

```diff
@@ -1,23 +1,32 @@
 package seedu.modtracker;
 
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
 /**
  * Main entry-point for the java.seedu.modtracker application.
  */
 public class ModTracker {
+    private Ui ui;
+    private ModuleList modList;
     private Storage storage;
-    private Ui ui = new Ui();
-    private ModuleList modList = new ModuleList();
 
     public static void main(String[] args) {
         new ModTracker().run();
     }
 
+    public ModTracker() {
+        ui = new Ui();
+        modList = new ModuleList();
+        storage = new Storage("data/modtracker.txt");
```

Comment:
I think the address can be refactored into a constant e.g. FILE_PATH

Reply:
Okay!